dist/apb.js

```diff
@@ -201,18 +198,42 @@ var apb = /** @class */ (function () {
             _a[stateName] = Object.assign({}, stateConfig, { Choices: choices }),
             _a;
     };
+    apb.prototype.generateParametersForSoclessTask = function (state_name, handle_state_kwargs) {
+        var parameters = {
+            "execution_id.$": "$.execution_id",
+            "artifacts.$": "$.artifacts",
+            "errors.$": "$.errors",
+            "results.$": "$.results",
+            "State_Config": {
+                "Name": state_name,
+                "Parameters": handle_state_kwargs
+            },
+        };
+        return parameters;
+    };
+    //! Interactions still need helper states ):
+    apb.prototype.generateParametersForSoclessInteraction = function (state_name, handle_state_kwargs, function_name) {
+        var parameters = {
+            FunctionName: function_name,
+            Payload: {
+                "sfn_context": this.generateParametersForSoclessTask(state_name, handle_state_kwargs),
+                "task_token.$": "$$.Task.Token",
+            },
+        };
+        return parameters;
+    };
     apb.prototype.transformTaskState = function (stateName, stateConfig, States, DecoratorFlags) {
-        var _a, _b;
+        var _a;
```

Comment:
we can fix this now or we can fix this later, but this needs a more descriptive name.